addons/bus/models/bus.py

```diff
@@ -100,7 +100,9 @@ def __init__(self):
         self.channels = {}
         self.started = False
 
-    def poll(self, dbname, channels, last, options=None, timeout=TIMEOUT):
+    def poll(self, dbname, channels, last, options=None, timeout=None):
+        if timeout is None:
+            timeout = TIMEOUT
```

Comment:
I don't see the point.  What difference does it make?

Reply:
> The ImDispatch.poll signature has been changed too so it is possible
to change (via code) the hardcoded default. The function was using the
object referenced by TIMEOUT at the time the function was defined,
using timeout None then if None: timeout=TIMEOUT ensure we lookup
the variable.

J'ai un module custom qui me sert à changer des trucs par-ci par-là, via ce module je veux pouvoir changer le timeout par défaut sans prendre le risque de changer la valeur dans bus.py et de la commiter par inadvertance